integrationTests/chainSimulator/mempool/testutils_test.go

```diff
@@ -402,10 +413,19 @@ func testFirstSelection(t *testing.T, sw *core.StopWatch, numTxs int, numTxsToBe
 
 	blockchainInfo := holders.NewBlockchainInfo([]byte("blockHash0"), 1)
 
+	f := createPprofFiles(t)
```

Comment:
I think we should explicitly call the start profiling & stop profiling commands not here (in the test utilities), but in the benchmark tests themselves. So that we can individually (for each benchmark) what we really want to measure).